clients/bigquery/dialect/dialect.go

```diff
@@ -15,9 +15,14 @@ import (
 	"github.com/artie-labs/transfer/lib/typing/ext"
 )
 
-const BQStreamingTimeFormat = "15:04:05"
+const (
+	srcAlias  = "cc"
+	destAlias = "c"
```

Comment:
Can we rename this:

srcAlias -> stagingAlias
destAlias -> targetAlias

Reply:
Sure.